docs/pages/api-reference/liveblocks-client.mdx

```diff
@@ -281,29 +281,33 @@ const unsubscribe = room.subscribe(root, (root) => {
 
 It’s also possible to subscribe to a storage item and its children by passing an
 optional `isDeep` parameter. In that case, the callback will get called with a
-list of updates instead. Each such update is a `{ type, node }` object.
+list of updates instead. Each such update is a `{ type, node, updates }` object.
 
 ```ts
 const { root } = await room.getStorage();
 const unsubscribe = room.subscribe(
   root,
-  (updates) => {
-    for (const update of updates) {
+  (storageUpdates) => {
+    for (const update of storageUpdates) {
       const {
-        type, // "update" or "delete"
+        type, // "LiveObject", "LiveList", or "LiveMap"
         node,
+        updates,
```

Comment:
Reading this, I was first slightly confused as to what `updates` was. But since it's different for each type, it became clear when reading the comments for each switch case below.